### PR TITLE
Add legacy CIC tournament articles

### PR DESCRIPTION
**Pull request by ReiFan49**
logo and podium (image) excluded for each tournaments (because either missing, expired, or never existed)

need some crosscheck before merge.

worknote:
- beatmap entry are processed manually and parsed based on one of tournament format (_I had to create a quickscript to fetch those_)
- user data, contestant order, staff data, and such are based from _manually mined_ CSV data, and its modified for efficiency.
- fun fact is optional (or even no right) right? (well i want but feel free to point it out)
- stub flag can be removed if it deemed to be a proper entry right

PS: hmu on discord (esp. indonesian contributors for further ID to EN crosschecks)

### CODE REVIEW

File: wiki/Tournaments/CIC/2012/en.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+  - [Nakagawa Shouko - Happily Ever After (dksslqj) \[Insane\]](https://osu.ppy.sh/beatmapsets/17550#fruits/62591)
+  - [Super Junior-M - Super Girl (Chinese Version) (JauiPlaY) \[Hard\]](https://osu.ppy.sh/beatmapsets/16084#fruits/90383)
+  - [sun3 - Higan Retour (saymun) \[Alazy\]](https://osu.ppy.sh/beatmapsets/14464#fruits/54371)
+  - [Kajiura Yuki - When Two Powers Collide (Lizbeth) \[Insane\]](https://osu.ppy.sh/beatmapsets/11864#osu/44983)
+- Tiebreaker
+  - **[Ryu\* - Bass 2 Bass (yangzd88) \[Another\]](https://osu.ppy.sh/beatmapsets/30296#osu/101727)**
+
+------------------------------------------------------------------------
+
+## Match Results
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:178]:
```suggestion
## Match results
```

File: wiki/Tournaments/CIC/2012/en.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+Due to old multiplayer data purged, multiplayer data can't be accessed.
+
+### Finals
+
+| Group | Saturday, 2012-07-28 |  |  |
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:183]:
```suggestion
| Group | Saturday, 2012-07-28 |  |  |
```
Don't know if this causes problem.
We usually use this format

```
28 July 2012:

| Group | Player 1 |  |  | Player 2 | Match link |
| :-: | --: | :-: | :-: | :-- | :-- |
| Championship | Player 1 | Player 1's score | Player 2's score | The match link |
```

File: wiki/Tournaments/CIC/2012/en.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+| :-: | --: | :-- | :-- |
+| Championship | ![][flag_ID] [nirvashzero](https://osu.ppy.sh/users/1254373) (0) | **![][flag_ID] [Rey-volution](https://osu.ppy.sh/users/883990) (3)** | --- |
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:185]:
```suggestion
| Championship | ![][flag_ID] [nirvashzero](https://osu.ppy.sh/users/1254373) (0) | **![][flag_ID] [Rey-volution](https://osu.ppy.sh/users/883990) (3)** | *missing* |
```
I think it's better to specify it missing with also italicised

File: wiki/Tournaments/CIC/2012/en.md [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+3. All the matches done in multiplayer mode, with following specification:
+    - Mode: CatchTheBeat
+    - Team: Team VS
+    - Win: Score
+4. Each participant should be on the room at most 10 minutes before the scheduled match time.
+5. "No Video" mod is the only mod that can be used.
+6. Room creation will be handed to the participants itself including the agreement between parties.
+7. After the match is concluded, player must hand the multiplayer link to the staffs or in the respective thread.
+
+[flag_ID]: /wiki/shared/flag/ID.gif
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:306]:
```suggestion
[flag_ID]: /wiki/shared/flag/ID.gif "Indonesia"
```
Reference naming

File: wiki/Tournaments/CIC/2012/en.md [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+| J | ![][flag_ID] [YuiKannon](https://osu.ppy.sh/users/1124510) |
+| K | ![][flag_ID] [r3yn41d0](https://osu.ppy.sh/users/500032) |
+| N | ![][flag_ID] [GenesisRogue](https://osu.ppy.sh/users/919004) |
+| P | ![][flag_ID] [Kenan Sisidi](https://osu.ppy.sh/users/518643) |
+
+------------------------------------------------------------------------
+
+## Ruleset
+
+### General Rules
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:281]:
```suggestion
### General rules
```
Sentence case thingy

File: wiki/Tournaments/CIC/2012/en.md [hunk 6]
```diff
@@ -0,0 +1,306 @@
...
+4. Staffs will handout the map pool that consisting **10 Maps and 1 Tie Breaker** 7 days before the round starts.
+5. Round will last up to 3 weeks. If it lasts shorter, the succeeding week will be the next round.
+6. If both party can't pick the schedule at most 3 days before the round ends, staffs will takeover the scheduling.
+7. Each paired participant should let the staff know about their match schedule.
+8. A match is considered valid as the staffs approves the schedule appointment.
+9. A match can be rescheduled at most 1 day before the current appointed time, if both participant and staff had approved it.
+10. Participant will be disqualified and eliminated if they don't show up after 10 minutes of the appointed time.
+11. If both participant had online on the match time and one of the staffs are online, they can ask the staffs directly to start the match.
+
+### Match Rules
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:294]:
```suggestion
### Match rules
```
Sentence case thingy

**Review by fajar13k — COMMENTED** [review]
Satu dulu bos

File: wiki/Tournaments/CIC/2012/en.md [hunk 7]
```diff
@@ -0,0 +1,306 @@
+---
+stub: true
+tags:
+- CIC 2012
+- CIC2012
+---
+
+# Catch the Beat Indonesia Cup 2012
+
+The **Catch the Beat Indonesia Cup 2012** (***CIC 2012***) is an Indonesian head-to-head osu!catch tournament hosted by ![][flag_ID] [Varri Naufalb](https://osu.ppy.sh/users/1174508). It is the 1st installment of the Catch the Beat Indonesia Cup.
```
[the remark this answers is not in the document]
Comment (ReiFan49) [reply wiki/Tournaments/CIC/2012/en.md:10]: yes it's 1v1

File: wiki/Tournaments/CIC/2012/en.md [hunk 8]
```diff
@@ -0,0 +1,306 @@
...
+
+## Participants
+
+| Group |  |  |  |  |
+| :-: | :-- | :-- | :-- | :-- |
+| **A** | ![][flag_ID] [swtboy](https://osu.ppy.sh/users/557944) | ![][flag_ID] [dOen9](https://osu.ppy.sh/users/1114648) |
+| **B** | ![][flag_ID] [Sheratan](https://osu.ppy.sh/users/960553) | ![][flag_ID] [pacmanmania](https://osu.ppy.sh/users/942688) |
+| **C** | ![][flag_ID] [Arashi354](https://osu.ppy.sh/users/897082) | ![][flag_ID] [moon\[light\]](https://osu.ppy.sh/users/478598) |
+| **D** | ![][flag_ID] [Sumatran](https://osu.ppy.sh/users/414409) | ![][flag_ID] [nirvashzero](https://osu.ppy.sh/users/1254373) |
+| **E** | ![][flag_ID] Intel21 | ![][flag_ID] akukadal2 |
```
[the remark this answers is not in the document]
Comment (ReiFan49) [reply wiki/Tournaments/CIC/2012/en.md:56]: well I assume it's the old link right?

File: wiki/Tournaments/CIC/2012/en.md (hunk 2 above)
Comment (ReiFan49) [reply wiki/Tournaments/CIC/2012/en.md:183]: I guess I have to generate a format for 1v1 as it's a generalized one (from CIC13)

File: wiki/Tournaments/CIC/2014/en.md [hunk 9]
```diff
@@ -0,0 +1,328 @@
...
+| Semifinals | 2014-06-27/2014-07-03 |
+| Finals | 2014-07-13/2014-07-13 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 4 months of osu!supporter tag |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter tag |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 months of osu!supporter tag |
```
Comment (ReiFan49) [inline wiki/Tournaments/CIC/2014/en.md:27]: holding this out until confirmation from ursa.

**Comment by TPGPL** [comment]
I added some commits to your PR, use `git pull` to synchronise the changes locally

**Comment by ReiFan49** [comment]
since I think there's no need to further hold it,
I leave the footnote on CIC 2014 one, gotta get the final check before merging, that's it.

File: wiki/Tournaments/CIC/2012/en.md [hunk 10]
```diff
@@ -0,0 +1,304 @@
...
+| :-- | :-- |
+| Staff | ![][flag_ID] [Varri Naufalb](https://osu.ppy.sh/users/1174508), ![][flag_ID] [-Newbie-](https://osu.ppy.sh/users/434914), ![][flag_ID] [awell](https://osu.ppy.sh/users/341298), ![][flag_ID] [Ableh\_ZZZ](https://osu.ppy.sh/users/917216), ![][flag_ID] [chibidanil](https://osu.ppy.sh/users/840925), ![][flag_ID] [AeroRange](https://osu.ppy.sh/users/405659), ![][flag_ID] [urs4chan](https://osu.ppy.sh/users/321063) |
+| Map selector | ![][flag_ID] [-Newbie-](https://osu.ppy.sh/users/434914), ![][flag_ID] [chibidanil](https://osu.ppy.sh/users/840925) |
+| Sponsor | ![][flag_ID] [chibidanil](https://osu.ppy.sh/users/840925), ![][flag_ID] [Varri Naufalb](https://osu.ppy.sh/users/1174508) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/74741)
+
+---
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:45]:
```suggestion
---
```
Preferably deleting all Line Breaks

File: wiki/Tournaments/CIC/2013/en.md [hunk 11]
```diff
@@ -0,0 +1,350 @@
+---
+tags:
+- CIC
+- CIC2013
+---
+
+# Catch the Beat Indonesia Cup 2013
+
+The **Catch the Beat Indonesia Cup 2013** (***CIC 2013***) was an Indonesian 4-players osu!catch tournament hosted by ![][flag_ID] [MOD-](https://osu.ppy.sh/users/1272422). It was the 2nd installment of the Catch the Beat Indonesia Cup.
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2013/en.md:9]:
```suggestion
The **Catch the Beat Indonesia Cup 2013** (***CIC 2013***) was an Indonesian 4-players osu!catch tournament hosted by ![][flag_ID] [MOD-](https://osu.ppy.sh/users/1272422). It was the second installment of the Catch the Beat Indonesia Cup.
```

File: wiki/Tournaments/CIC/2014/en.md [hunk 12]
```diff
@@ -0,0 +1,355 @@
+---
+tags:
+- CIC
+- CIC2014
+---
+
+# Catch the Beat Indonesia Cup 2014
+
+The **Catch the Beat Indonesia Cup 2014** (***CIC 2014***) is an Indonesian 4-players osu!catch tournament hosted by ![][flag_ID] [urs4chan](https://osu.ppy.sh/users/321063). It is the 3rd installment of the Catch the Beat Indonesia Cup.
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2014/en.md:9]:
```suggestion
The **Catch the Beat Indonesia Cup 2014** (***CIC 2014***) is an Indonesian 4-players osu!catch tournament hosted by ![][flag_ID] [urs4chan](https://osu.ppy.sh/users/321063). It is the third installment of the Catch the Beat Indonesia Cup.
```

File: wiki/Tournaments/CIC/2014/en.md [hunk 13]
```diff
@@ -0,0 +1,355 @@
...
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2014-03-15/2014-03-30 |
+| Knockout | 2014-06-06/2014-06-08 |
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2014/en.md:16]:
```suggestion
| Knockout Stage | 2014-06-06/2014-06-08 |
```
Works better imo

File: wiki/Tournaments/CIC/2014/en.md [hunk 14]
```diff
@@ -0,0 +1,355 @@
...
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 4 months of osu!supporter tag |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter tag |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag |
+
+Note: *Still looking for confirmation of prize pool.*
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2014/en.md:29]:
```suggestion
*Note: Still looking for confirmation of prize pool.*
```

**Review by fajar13k — COMMENTED** [review]
minors

File: wiki/Tournaments/CIC/2014/en.md (hunk 13 above)
Comment (ReiFan49) [reply wiki/Tournaments/CIC/2014/en.md:16]: wait not sentence? 🤔

File: wiki/Tournaments/CIC/2014/en.md [hunk 15]
```diff
@@ -0,0 +1,345 @@
+---
+tags:
+- CIC
+- CIC2014
+---
+
+# Catch the Beat Indonesia Cup 2014
+
+The **Catch the Beat Indonesia Cup 2014** (***CIC 2014***) is an Indonesian 4-players osu!catch tournament hosted by ![][flag_ID] [urs4chan](https://osu.ppy.sh/users/321063). It is the third installment of the Catch the Beat Indonesia Cup.
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2014/en.md:9]:
```suggestion
The **Catch the Beat Indonesia Cup 2014** (***CIC 2014***) was an Indonesian 4-players osu!catch tournament hosted by ![][flag_ID] [urs4chan](https://osu.ppy.sh/users/321063). It was the third installment of the Catch the Beat Indonesia Cup.
```

File: wiki/Tournaments/CIC/2014/en.md [hunk 16]
```diff
@@ -0,0 +1,345 @@
...
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 4 months of osu!supporter tag |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter tag |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag |
+
+*Note: Still looking for confirmation of prize pool.*
+
+## Organisation
+
+The Catch the Beat Indonesia Cup 2014 is run by various Indonesian osu!catch community members.
```
Comment (fajar13k) [inline wiki/Tournaments/CIC/2014/en.md:33]:
```suggestion
The Catch the Beat Indonesia Cup 2014 was run by various Indonesian osu!catch community members.
```

**Review by fajar13k — COMMENTED** [review]
Very minor ;;;;;

**Review by TPGPL — APPROVED** [review]

**Pull request merged**